samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -50,7 +50,8 @@ enum link_shell_command {
 	LINK_CMD_DNSADDR,
 	LINK_CMD_REDMOB,
 	LINK_CMD_PROPRIPSM,
-	LINK_CMD_MODEM
+	LINK_CMD_MODEM,
+	LINK_CMD_PALL,
```

Comment:
The implementation has of course already done, but I'm just wondering what is the benefit of having this functionality in modem_shell. The command format when writing the list using the new command is not that different from the AT command, at least the PLMN/access technology list part.

Reply:
Yeah, we just had a brief discussion about this on Teams and it seems that there might not even be an actual need of this functionality at such a high level API...